pkg/types/baremetal/validation/platform.go

```diff
@@ -37,12 +41,31 @@ func ValidatePlatform(p *baremetal.Platform, fldPath *field.Path) field.ErrorLis
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("apiVIP"), p.APIVIP, err.Error()))
 	}
 
+	if n != nil {
```

Comment:
> https://github.com/openshift/installer/blob/1dbe72c360f80a214544f85d10e7747d34837a09/pkg/types/baremetal/validation/platform.go#L40-L68

can we try to DRY this up a little with a local function??

Reply:
Sure thing.